docs/src/organisms/titleBar.js

```diff
@@ -122,77 +122,71 @@ export default class ViewsTitleBar extends React.Component {
+                    <Header size="large" style={{ marginTop: '55px' }} sub>
                         Title Bar
                         <Header.Subheader>
                             A title bar is a wrapper for the a page title. It is to be placed at the very top of each page according to the design.
                         </Header.Subheader>
                     </Header>
 
                     <MediaQuery maxWidth={767}>
-                        {matches => {
-                            return (
-                                <Card style={{ padding: matches ? '0 11px' : '0 22px' }}>
-                                    <TitleBar title="The Best Title Bar" />
-                                </Card>
-                            );
-                        }}
+                        {(matches) => (
+                            <Card style={{ padding: matches ? '0 11px' : '0 22px' }}>
+                                <TitleBar title="The Best Title Bar" />
+                            </Card>
+                        )}
                     </MediaQuery>
 
                     <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
                         {titleBarSample}
                     </Highlighter>
 
                     {/* Sub Title */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub={true}>
+                    <Header size="large" style={{ marginTop: '55px' }} sub>
                         Sub Title
                         <Header.Subheader>
                             Title Bar's can have a sub title.
                         </Header.Subheader>
                     </Header>
 
                     <MediaQuery maxWidth={767}>
-                        {matches => {
-                            return (
-                                <Card style={{ padding: matches ? '0 11px' : '0 22px' }}>
-                                    <TitleBar subTitle="The Best Title Bar" title="An Even Better Title" />
-                                </Card>
-                            );
-                        }}
+                        {(matches) => (
+                            <Card style={{ padding: matches ? '0 11px' : '0 22px' }}>
+                                <TitleBar subTitle="The Best Title Bar" title="An Even Better Title" />
+                            </Card>
+                        )}
                     </MediaQuery>
 
                     <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
                         {subTitleSample}
                     </Highlighter>
 
                     {/* Children */}
-                    <Header size="large" style={{ marginTop: '55px' }} sub={true}>
+                    <Header size="large" style={{ marginTop: '55px' }} sub>
                         Children
                         <Header.Subheader>
                             Title Bar's can return it's children on the right side of the bar.
```

Comment:
Grammar nitpick: "its" (no apostrophe) is the possessive form of "it".  "it's" with an apostrophe is a contraction for "it is".
https://www.grammarly.com/blog/its-vs-its/

Reply:
Probably don't need "'s" after "Title Bar" either (Should be "Title Bar can return its children..." versus "Title Bar's can return it's children...") ... all sorts of bad grammar there! 😁